src/Controllers/Common/ContentController.cs

```diff
@@ -778,9 +778,16 @@ public IActionResult GetAvatarByUserId([FromForm] Guid userId)
 
     [HttpPost]
     [Produces("application/xml")]
-    [Route("ContentWebService.asmx/GetUnselectedPetByTypes")] // used by old SoD (e.g. 1.13)
+    [Route("ContentWebService.asmx/GetUnselectedPetByTypes")] // used by old SoD (e.g. 1.13) and Math Blaster
```

Comment:
no need add `and Math Blaster` here ... `// used by X` means endpoint is (or may be) used by any game with `gameVersion` / value returned by `ClientVersion.GetVersion(apiKey)` lower equal than value for X ... I'm 99% sure this one is also used by WoJS for example